portality/view/doaj.py

```diff
@@ -427,127 +427,127 @@ def google_webmaster_tools():
 
 @blueprint.route("/accessibility/")
 def accessibility():
-    return render_template("layouts/static_page.html", page_frag="/accessibility-fragment.html")
+    return render_template("layouts/static_page.html", page_frag="/accessibility-fragment.html", page_title="Accessibility statement for doaj.org")
```

Comment:
I can see that adding these to the templates necessitates passing in the `page_title` - but I don't like that string constants have been added to the code. Perhaps we should have a lookup from config for these page titles.

Reply:
I don’t know how to set that up :\

Reply:
not a problem, we can bring it up in the meeting. `doaj.py` does have almost all of the static page routes, so maybe it's okay to keep them here for now. Otherwise it'd be something like a yaml file with all of the titles.